<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DACCalibrationClass</class>
 <widget class="QWidget" name="DACCalibrationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DACCalibration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:rgba(0,0,0,0);
}</string>
  </property>
  <widget class="QGroupBox" name="CalibBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>2</y>
     <width>391</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 10px;
     margin-top: 1ex; /* leave space at the top for the title */
	font:18px bold;
	color:black;
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
	 background-color:white;
     border:2px solid black;
	border-radius:10px;
 }</string>
   </property>
   <property name="title">
    <string>VI DRIVE/RECEIVE CALIBRATION PANEL</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>No of Points:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 8px;
     padding: 0 8px;
     background: white;
     selection-background-color: darkgray;
	 color:black;
     font: bold 12px;
 }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Drive Range:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
     border: 2px solid gray;
     border-radius: 5px;
     padding: 1px 18px 1px 3px;
	background-color:white;
	color:black;
 }
</string>
    </property>
    <item>
     <property name="text">
      <string>0.6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Drive Volt(x):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 8px;
     padding: 0 8px;
     background: white;
     selection-background-color: darkgray;
	 color:black;
     font: bold 12px;
 }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="driveBut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 10px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
    </property>
    <property name="text">
     <string>DRIVE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RX'd Volt(y):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 8px;
     padding: 0 8px;
     background: white;
     selection-background-color: darkgray;
	 color:black;
     font: bold 12px;
 }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 10px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
    </property>
    <property name="text">
     <string>NEXT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Slope:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="slope">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>90</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 8px;
     padding: 0 8px;
     background: gray;
     selection-background-color: darkgray;
	 color:white;
     font: bold 12px;
 }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>190</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Constant:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="constant">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>90</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 8px;
     padding: 0 8px;
     background: gray;
     selection-background-color: darkgray;
	 color:white;
     font: bold 12px;
 }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBut">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 10px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBut">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 10px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>42</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>244</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test Volt:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>244</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 8px;
     padding: 0 8px;
     background: white;
     selection-background-color: darkgray;
	 color:black;
     font: bold 12px;
 }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="driveBut_2">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>240</y>
      <width>84</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 10px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
    </property>
    <property name="text">
     <string>DRIVE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Calibrated">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>93</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calibrated</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>220</y>
      <width>382</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 1px solid gray;
     border-radius: 10px;
     margin-top: 1ex; /* leave space at the top for the title */
	font:18px bold;
	color:black;
 }

 </string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>108</x>
       <y>60</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 8px;
     padding: 0 8px;
     background: white;
     selection-background-color: darkgray;
	 color:black;
     font: bold 12px;
 }</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rxed Volt:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="driveBut_3">
     <property name="geometry">
      <rect>
       <x>294</x>
       <y>60</y>
       <width>84</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 10px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>217</x>
       <y>60</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
     border: 2px solid gray;
     border-radius: 5px;
     padding: 1px 18px 1px 3px;
	background-color:white;
	color:black;
 }
</string>
     </property>
     <item>
      <property name="text">
       <string>0.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>117</x>
       <y>88</y>
       <width>80</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="samples">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>144</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="CalibratedRef">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>343</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calibrated Reference</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_2</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>driveBut</zorder>
   <zorder>label_4</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>nextBut</zorder>
   <zorder>label_5</zorder>
   <zorder>slope</zorder>
   <zorder>label_6</zorder>
   <zorder>constant</zorder>
   <zorder>saveBut</zorder>
   <zorder>exitBut</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>driveBut_2</zorder>
   <zorder>Calibrated</zorder>
   <zorder>samples</zorder>
   <zorder>CalibratedRef</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
